app/views/courses/show_professors.html.slim

```diff
@@ -42,7 +42,7 @@
                     span GPA
                   .row.course-rating-row
                     - if stats.gpa
-                      h4.course-rating = stats.gpa.round(2)
+                      h4.course-gpa = stats.gpa.round(2)
```

Comment:
Is there any reason you're setting them to separate classes? What do you think about having a single css *class* for these rating fields. Then using an id for the javascript side of things?